src/controllers/AbstractController.ts

```diff
@@ -164,6 +164,26 @@ export default abstract class AbstractController<T extends AbstractService> {
 		return num;
 	}
 
+	protected parseQueryParamArrayOrThrow(
+		n: string,
+		errorMessage: string
+	): number[] {
+		const isValidParam = /[^,]+/.test(n);
```

Comment:
Yea I also realized this regex is right but wrong. It will pass for all cases basically which is not what I want. Ex: `aaa,aaa` => true `aaa,,,,a`  => true (should be false), and there are many more cases to, Going to rewrite this one.

Which is also a good reminder this method needs a test to confirm that.

Reply:
Its of trying to hand roll our own regex what if we used the express syntax for query param arrays? See http://expressjs.com/en/api.html#req.query

Reply:
Yea i considered it at first, and now that you mention it again I think it's probably the best route we have considering my lack of regex knowledge. Will change it to reflect the built in query support